<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>个人档案照片处理工具</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="imageLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_width">
            <property name="text">
             <string>宽</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="widthLineEdit">
            <property name="text">
             <string>358</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>px</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_high">
            <property name="text">
             <string>高</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="heightLineEdit">
            <property name="text">
             <string>441</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>px</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_resolution">
            <property name="text">
             <string>分辨率</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="resolutionLineEdit">
            <property name="text">
             <string>350</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>dpi</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="sizeLimitLabel">
            <property name="text">
             <string>大小限制</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="sizeLineEdit">
            <property name="text">
             <string>50</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>kb</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>选择照片</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportButton">
        <property name="text">
         <string>导出照片</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
